docs/en/observability/logs-stream.asciidoc

```diff
@@ -235,29 +235,24 @@ When you ingested the document in the previous section, you'll notice the `@time
         ...
 ----
 
-This section shows you how to extract the `@timestamp` field from the example log so you can filter by when logs occurred and when issues happened.
+This section shows you how to extract the `@timestamp` field from the log message, so you can filter by when the logs and issues actually occurred.
 
-[source,log]
-----
-2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%.
-----
+To extract the timestamp, you need to:
 
-To extract the timestamp you need to:
-
-- <<logs-stream-ingest-pipeline>>
-- <<logs-stream-simulate-api>>
-- <<logs-stream-index-template>>
-- <<logs-stream-create-data-stream>>
+. <<logs-stream-ingest-pipeline>>
+. <<logs-stream-simulate-api>>
+. <<logs-stream-index-template>>
+. <<logs-stream-create-data-stream>>
 
 [discrete]
 [[logs-stream-ingest-pipeline]]
 === Use an ingest pipeline to extract the `@timestamp`
 
-To extract the `@timestamp` field from the example log, use an ingest pipeline with a dissect processor. Ingest pipelines in {es} are used to process incoming documents. The {ref}/dissect-processor.html[dissect processor] is one of the available processors that extracts structured fields from your unstructured log message based the pattern you set. In the following example command, the dissect processor extracts the timestamp to the `@timestamp` field.
+Ingest pipelines consist of a series of processors that perform common transformations on incoming documents before they are indexed. To extract the `@timestamp` field from the example log, you can use an ingest pipeline with a dissect processor. The {ref}/dissect-processor.html[dissect processor] extracts structured fields from unstructured log messages based on a pattern you set. 
```

Comment:
I could be wrong about this. @dedemorton will know.

I wouldn't say:

> To extract the `@timestamp` field from the example log, **you can** use an ingest pipeline with a dissect processor.

I would say:

> To extract the `@timestamp` field from the example log, use an ingest pipeline with a dissect processor.

Reply:
I think you're right here, and this may have been a blind spot in my writing. 

At least for Microsoft:

> When you see can in your writing, consider deleting it. Can implies ability but not action. Rewrite to describe the action if possible. When ability is what you need to express, it's OK to use can to describe actions or tasks that the reader or program is able to do.

I'm putting this can in the trash can.

Reply:
Microsoft said it much better than I ever could have.